src/org/labkey/test/components/react/ReactDatePicker.java

```diff
@@ -37,10 +37,16 @@ public String get()
         return elementCache().input.get();
     }
 
-    public void set(String value)
+    public void set(String value, boolean close)
     {
         elementCache().input.set(value);
-        elementCache().input.getComponentElement().sendKeys(Keys.ENTER); // Dismiss date picker
+        if (close)
+            elementCache().input.getComponentElement().sendKeys(Keys.ENTER); // Dismiss date picker
+    }
+
+    public void set(String value)
+    {
+        set(value, true);
     }
```

Comment:
Create a new method for setting via the UI
```suggestion
        elementCache().input.getComponentElement().sendKeys(Keys.ENTER); // Dismiss date picker
    }

    public void setWithPopup(String yearMonth, String day, String time)
    {
        expand();
        elementCache().input.set(value); // use keyboard input to set year and month
        elementCache().datePickerDateCell(day).click(); // use calendar ui to select day
        if (!StringUtils.isEmpty(time)) // use timepicker to select time
            clickTime(time); // Clicking a time dismisses the date picker
        else
            elementCache().input.getComponentElement().sendKeys(Keys.ENTER); // Dismiss date picker
    }
```